src/plays/personal-profile-card/components/profile-form.tsx

```diff
@@ -0,0 +1,138 @@
+          onChange={onChange}
+          value={value ? value.name : ""}
+        />
+        <textarea
+          name="description"
+          rows={4}
+          className="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500 mb-2 placeholder-gray-400"
+          placeholder="@your description"
+          onChange={onChange}
+          value={value ? value.description : ""}
+        ></textarea>
+      </div>
+      <div className="social-links">
+        <input
+          value={value ? value.facebook : ""}
+          type="text"
+          name="facebook"
+          className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 mb-2"
+          placeholder="@facebook handle"
+          required
+          onChange={onChange}
+        />
+        <input
+          value={value ? value.twitter : ""}
+          type="text"
+          name="twitter"
+          className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 mb-2"
+          placeholder="@twitter handle"
+          required
+          onChange={onChange}
+        />
+        <input
+          value={value ? value.instagram : ""}
+          type="text"
+          name="instagram"
+          className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 mb-2"
+          placeholder="@instagram handle"
+          required
+          onChange={onChange}
+        />
+        <input
+          value={value ? value.github : ""}
+          type="text"
+          name="github"
+          className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 mb-2"
+          placeholder="@github handle"
+          required
+          onChange={onChange}
+        />
+        <input
+          value={value ? value.linkedin : ""}
+          type="text"
+          name="linkedin"
+          className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 mb-2"
+          placeholder="@linkedin handle"
+          required
+          onChange={onChange}
+        />
+        <button
+          type="button"
+          className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 focus:outline-none"
+          onClick={onClick}
+        >
+          {profile === null ? "Save Profile Card" : "Update Profile Card"}
+        </button>
+        <button
+          type="button"
+          className="text-white bg-red-700 hover:bg-red-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 focus:outline-none"
+          disabled={profile === null && true}
```

Comment:
Add this condition to clear only the already saved profile. But I think it is not needed. Because better to give option to clear at any time. So remove the `disabled={profile === null && true} `